frontend_lib/src/component/PopinFixed/PopinFixed.jsx

```diff
@@ -2,37 +2,64 @@ import React from 'react'
 import classnames from 'classnames'
 import PropTypes from 'prop-types'
 import PopinFixedHeader from './PopinFixedHeader.jsx'
-import PopinFixedOption from './PopinFixedOption.jsx'
 import PopinFixedContent from './PopinFixedContent.jsx'
+import { CUSTOM_EVENT } from '../../customEvent.js'
+import { TracimComponent } from '../../tracimComponent.js'
 
 // require('./PopinFixed.styl') // see https://github.com/tracim/tracim/issues/1156
 
-const PopinFixed = props => {
-  return (
-    <div
-      className={classnames('wsContentGeneric', props.customClass, { visible: props.visible })}
-      style={props.style}
-      data-cy='popinFixed'
-    >
-      {props.children}
-    </div>
-  )
-}
+class PopinFixed extends React.Component {
+  constructor (props) {
+    super(props)
+
+    this.state = {
+      isSidebarVisible: true
```

Comment:
Is the sidebar always visible when this component is created?

Reply:
No, this is a known bug that we have decided to address in the next sprint.

Note: If you have an idea how to do this detection, don't hesitate to tell us.

Reply:
Without having thought much about it, I would put this state in redux, manage its changes in the sidebar component and communicate it to apps like we already do for some other redux states.

Reply:
Not really, I did not take the time to think about the problem. Thx for the idea.